tests/conftest.py

```diff
@@ -30,7 +30,7 @@
 TOL = 1e-3
 CUTOFF = 6
 ALPHA = 0.1
-HBAR = 2
+HBAR = 1.7
```

Comment:
```suggestion
HBAR = 2
```

Reply:
Was this set back to 2 in merged version?